docs/ui/storage/fragments/react-native/s3-image.md

```diff
@@ -1,82 +1,40 @@
 `S3Image` component renders an *Amazon S3 object key* as an image:
 
 ```jsx
```

Comment:
Lets try to render an image on both ios and android simulator and update it here.